pennylane/qnode.py

```diff
@@ -148,15 +148,192 @@ class QNode:
     >>> @qml.qnode(dev)
     ... def circuit(x):
     ...     qml.RX(x, wires=0)
-    ...     return expval(qml.PauliZ(0))
+    ...     return qml.expval(qml.PauliZ(0))
 
     or by instantiating the class directly:
 
     >>> def circuit(x):
     ...     qml.RX(x, wires=0)
-    ...     return expval(qml.PauliZ(0))
+    ...     return qml.expval(qml.PauliZ(0))
     >>> dev = qml.device("default.qubit", wires=1)
     >>> qnode = qml.QNode(circuit, dev)
+
+    .. details::
+        :title: Parameter broadcasting
+        :href: parameter-broadcasting
+
+        QNodes can be executed simultaneously for multiple parameter settings, which is called
+        *parameter broadcasting* or *parameter batching*.
+        We start with a simple example and briefly look at the scenarios in which broadcasting is
+        possible and useful. Finally we give rules and conventions regarding the usage of
+        broadcasting, together with some more complex examples.
+        Also see the :class:`~.pennylane.operation.Operator` documentation for implementation
+        details.
+
+        **Example**
+
+        Consider the ``circuit`` defined above. If we want to execute it at multiple values of
+        ``x``, we may pass those as a one-dimensional array to the QNode:

```

Comment:
@dwierichs if the reader has executed all codeblocks in order above, then `circuit` would not be a qnode :(

Reply:
Oh right, so I just want to write ``qnode`` instead of ``circuit``, correct? :)

Reply:
Or maybe better to just define a qnode/circuit here.